Test/PerformanceCollector/FunctionalTests/Test40.cs

```diff
@@ -38,7 +39,15 @@ public void TestInitialize()
                     IKey = "fafa4b10-03d3-4bb0-98f4-364f0bdf5df8",
                 });
 
-            base.LaunchAndVerifyApplication();
+            try
+            {
+                base.LaunchAndVerifyApplication();
+            }
+            catch (Exception ex)
+            {
+                Trace.WriteLine("Exception occured while verifying application, exception:" + ex.Message);
+                this.StopWebAppHost();
```

Comment:
You are right. I'll throw the ex , so that tests won't be run at all since failure is at test initialize. Good catch!